### ISSUE TITLE
Includes - adding fields to be saved by the form

### ISSUE THREAD

**Issue by rouxxx** [darklow/django-suit#465]
Hi all,

I am using several includes in my project and everything works well so far.

For a screen, I need to customize a tab with a special form design. So I used an include to insert the design I want. This include contains fields that need to be saved like the other fields of the form.
If I add the input in raw HTML, they appear but they are not saved.
If I try to use field tags, nothing appears.

One point, I can access the model data with `{{ original.field }}` but it's just a readonly value.

Is what I want to do possible and am I missing anything ?

Suit 0.2.15
Django 1.7.8

**Comment by darklow** [comment]
Maybe example would help, otherwise it is hard to follow on what you are doing.
And what do you mean by accessing field as readonly - I think you should not do anything more than reading data in templates. Writing data from templates are wrong.

**Comment by rouxxx** [comment]
I'll explain it better.

My need is inside an admin form and customize a part of the form using an include.

![custom_form](https://cloud.githubusercontent.com/assets/12110493/12288326/80fa1636-b9d3-11e5-99ee-906872c81dc4.PNG)

The other tabs are automatically generated like any admin form.

Thanks !

**Comment by darklow** [comment]
Ok, this is much more complicated and this issue is not related to Django Suit actually.
Django Admin accepts and saves only fields that are included in the AdminForm.
The fact that you rendered custom form doesn't mean anything.
If you are using your own form you will have to handle, validate and save it by yourself.
Easiest way should be extending Django Admin Form.

You can take a look at these links for some ideas:
https://docs.djangoproject.com/en/1.7/ref/contrib/admin/#adding-custom-validation-to-the-admin
http://stackoverflow.com/questions/17948018/django-admin-add-custom-form-fields-that-are-not-part-of-the-model

**Comment by rouxxx** [comment]
great thank you !

That's what I was thinking, I just hoped that includes could save me ;)

**Comment by rouxxx** [comment]
just to let you know, thanks to your help i managed to do it.

I think it would be a great help for suit users to have a small paragraph in the documentation about that. Using includes can help you to partially customize your admin forms easily.

Once again, thank you !

**Comment by darklow** [comment]
Glad you solved your issue!
